### PR TITLE
[pt-br] fix content/pt-br/docs/concepts/scheduling-eviction/pod-overhead.md

### PR DESCRIPTION
**Pull request by DonatoHorn**
This is one of the pages I found, the contents are from the v1.18.

/assign sftim

### CODE REVIEW

**Comment by sftim** [comment]
/unassign

I'm a tech lead for SIG Docs but I'm not a reviewer for Portuguese.

File: content/pt-br/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 1]
```diff
@@ -2,43 +2,41 @@
 title: Sobrecarga de Pod 
 content_type: concept
 weight: 50
+update_date: 2022-07-22
+origin_version: 1.24
+contributors:
+- DonatoHorn
+reviewers:
```
Comment (sftim) [inline content/pt-br/docs/concepts/scheduling-eviction/pod-overhead.md:9]: I'm really not keen on seeing these lines added. They aren't a convention that we've accepted within SIG Docs.

Comment (DonatoHorn) [reply content/pt-br/docs/concepts/scheduling-eviction/pod-overhead.md:9]: Sorry, understand.

**Comment by sftim** [comment]
439 commits? Doesn't look right.

/hold
Other contributors: please only unhold once any problems so far with this PR are _all_ clear.

**Comment by tengqm** [comment]
Given the status of the PR, I'd close it so that the author can open a clean one.
/close

**Comment by jihoon-seo** [comment]
/remove-area blog
/remove-language de en es ja ko pl zh

**Pull request closed without merging**